openslides_backend/action/action_handler.py

```diff
@@ -17,11 +17,20 @@
     Payload,
 )
 
+action_payload_schema = {
+    "$schema": schema_version,
+    "title": "Action payload",
+    "type": "array",
+    "items": {"type": "object"},
+    "minItems": 1,
+    "uniqueItems": True,
```

Comment:
This is not needed. I removed it in a previous PR from the other schemas. It should be possible to just give an empty list of payloads - then nothing happens. Also, the items don't have to be unique. Imagine a case where you want for some reason to create the same object (with the same content) multiple times. It doesn't matter if this makes sense but we should not forbid it.

Reply:
Fixed.